ts/Core/Chart/Chart.ts

```diff
@@ -2324,26 +2324,37 @@ class Chart {
 
         // Apply new links
         chartSeries.forEach(function (series): void {
-            let linkedTo = series.options.linkedTo;
+            const { linkedTo } = series.options;
 
-            if (isString(linkedTo)) {
+            if (linkedTo && isString(linkedTo)) {
```

Comment:
Do we need the extended checked for `if (linkedTo` ? This will fail if `linkedTo` is an empty string, but is it necessary to change how it was?